graph/graph.go

```diff
@@ -112,11 +113,18 @@ func NewGraph(root string, driver graphdriver.Driver) (*Graph, error) {
 	}
 
 	graph := &Graph{
-		root:     abspath,
-		idIndex:  truncindex.NewTruncIndex([]string{}),
-		driver:   driver,
-		retained: &retainedLayers{layerHolders: make(map[string]map[string]struct{})},
+		root:             abspath,
+		idIndex:          truncindex.NewTruncIndex([]string{}),
+		driver:           driver,
+		retained:         &retainedLayers{layerHolders: make(map[string]map[string]struct{})},
+		tarSplitDisabled: false,
 	}
+
+	// Windows does not currently support tarsplit functionality.
+	if runtime.GOOS == "windows" {
+		graph.tarSplitDisabled = true
```

Comment:
No, it's at the registry key level, basically a union FS for the registry. But we store all the changed and added keys (with some special keys for tombstones) in a single opaque registry hive file (well, hive file per registry hive -- there are several per container).

In theory we could treat the registry like any other file and copy the whole thing on write, but it's quite large and will be modified for every container startup, so we think this is a worthwhile optimization.